lib/errorHandlers/suppressError.ts

```diff
@@ -59,7 +77,10 @@ function createPlatformVersionError(err, subCategory) {
   uiLine();
 }
 
-function shouldSuppressError(err, context = {}) {
+export function shouldSuppressError(
+  err: unknown,
+  context?: typeof ApiErrorContext
```

Comment:
We have an `ApiErrorContext` type in LDL.  I wonder if this would be a good time to abandon the class `ApiErrorContext` in favor of the type.

Reply:
@joe-yeager, do you mean `HubSpotHttpErrorContext` here? There's no error type explicitly called `ApiErrorContext`.

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1251/commits/146dcbafeabd6d865cb305cb61cf867207c31fbf